spring-boot-reactive-web/src/main/java/com/thepracticaldeveloper/reactiveweb/domain/Quote.java

```diff
@@ -1,59 +1,17 @@
 package com.thepracticaldeveloper.reactiveweb.domain;
 
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
```

Comment:
@Data annotation already has these implemented, i.e. if you drop all other annotations, it will work fine!
Also you can add the @Builder annotation to use builder pattern